components/dashboard/src/menu/Menu.tsx

```diff
@@ -52,64 +47,49 @@ export default function Menu() {
         ]).then((setters) => setters.forEach((s) => s()));
     }, [setCurrency, setIsChargebeeCustomer, setIsStudent]);
 
-    const leftMenu: Entry[] = [
-        {
-            title: "Workspaces",
-            link: "/workspaces",
-            alternatives: ["/"],
-        },
-        {
-            title: "Projects",
-            link: `/projects`,
-            alternatives: [] as string[],
-        },
-    ];
-
-    const adminMenu: Entry = {
-        title: "Admin",
-        link: "/admin",
-        alternatives: [
-            ...getAdminTabs().reduce(
-                (prevEntry, currEntry) =>
-                    currEntry.alternatives
-                        ? [...prevEntry, ...currEntry.alternatives, currEntry.link]
-                        : [...prevEntry, currEntry.link],
-                [] as string[],
-            ),
-        ],
-    };
+    const adminMenu: Entry = useMemo(
+        () => ({
+            title: "Admin",
+            link: "/admin",
+            alternatives: [
+                ...getAdminTabs().reduce(
+                    (prevEntry, currEntry) =>
+                        currEntry.alternatives
+                            ? [...prevEntry, ...currEntry.alternatives, currEntry.link]
+                            : [...prevEntry, currEntry.link],
+                    [] as string[],
+                ),
+            ],
+        }),
+        [],
+    );
 
-    const handleFeedbackFormClick = () => {
+    const handleFeedbackFormClick = useCallback(() => {
         setFeedbackFormVisible(true);
-    };
+    }, []);
 
-    const onFeedbackFormClose = () => {
+    const onFeedbackFormClose = useCallback(() => {
         setFeedbackFormVisible(false);
-    };
+    }, []);
 
     return (
         <>
-            <header className="app-container flex flex-col pt-4 space-y-4" data-analytics='{"button_type":"menu"}'>
-                <div className="flex h-10 mb-3">
-                    <div className="flex justify-between items-center pr-3">
-                        <Link to="/" className="pr-3 w-10">
+            <header className="app-container flex flex-col pt-4" data-analytics='{"button_type":"menu"}'>
+                <div className="flex justify-between h-10 mb-3 w-full">
+                    <div className="flex items-center">
+                        {/* hidden on smaller screens */}
+                        <Link to="/" className="hidden md:inline pr-3 w-10">
```

Comment:
**thought**: Seems like there could be room for keeping the logo but let's try this and see if we need to revert this.